pkg/api/storage/storage.go

```diff
@@ -38,6 +38,28 @@ func (c *Client) ListBucket(ctx context.Context, opts *contracts.ListOptions) (*
 	return resp, nil
 }
 
+func (c *Client) DeleteBucket(ctx context.Context, name string) error {
+	logger.Debug("Delete bucket")
+	_, httpResp, err := c.apiClient.StorageAPI.
+		StorageApiBucketsDestroy(ctx, name).Execute()
+	if err != nil {
+		logger.Error("Error while listing buckets", zap.Error(err))
```

Comment:
I'm no `Go` _connoisseur_ so I might not completely understand what is going on here but... Is the error really `listing` buckets or should it be `deleting the bucket`? 🤔

Reply:
thanks, the message is wrong